EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -678,7 +687,10 @@ void HcalUnpacker::unpackUTCA(const FEDRawData& raw, const HcalElectronicsMap& e
 
 	// Insert data
     /////////////////////////////////////////////CODE FROM OLD STYLE DIGIS///////////////////////////////////////////////////////////////
-	if (!did.null()) { // unpack and store...
+	if (!did.null() && did.det()==DetId::Calo && did.subdetId()==HcalZDCDetId::SubdetectorId) { // unpack and store...
```

Comment:
This is where the `unpackZDCQIE10_` flag from `HcalRawToDigi` should be propagated in order not to be bogus.